deepcell_label/label.py

```diff
@@ -50,11 +50,16 @@ class Edit(object):
     TODO: modify MutableNdarray class to share changed() signals from arrays view
     """
 
-    def __init__(self, labels, raw=None):
+    def __init__(self, labels, raw=None, overlaps=None, write_mode='overlap'):
```

Comment:
The valid options are `overlap` `exclude` and `overwrite`, I'll add them to the docstring

Reply:
The constructor changes in the next PR, so I am going to make docstring changes so I don't change it twice

Reply:
Here's the docstring I wrote for the next PR
```
class Edit(object):
    """
    Loads labeled data from a zip file,
    edits the labels according to edit.json in the zip,
    and writes the edited labels to a new zip file.

    The labels zipfile must contain:
        labeled.dat - a binary array buffer of the labeled data (int32)
        overlaps.json - a 2D json array describing values encode which cells
                        the (i, j)th element of overlaps.json is 1 if value i encodes cell j and 0 otherwise
        edit.json - a json object describing the edit to be made including
                    - action (e.g. )
                    - the args for the action
                    - write_mode: one of 'overlap', 'overwrite', or 'exclude'
                    - height: the height of the labeled (and raw) arrays
                    - width: the width of the labeled (and raw) arrays
    It additionally may contain:
        raw.dat - a binary array buffer of the raw data (uint8)
        lineage.json - a json object describing the lineage of the cells
    """
```